resources/upgrading-to-aep/current-sdk-versions.md

```diff
@@ -65,9 +65,9 @@ The Swift iOS SDKs are open source - read more about [our move to Swift and open
 
 {% endhint %}
 
-| Extension | CocoaPods | GitHub |
-| :--- | :--- | :--- |
-| [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) | [![Cocoapods](https://img.shields.io/cocoapods/v/AEPCore.svg?color=orange&label=AEPCore&logo=apple&logoColor=white&style=flat-square)​](https://cocoapods.org/pods/AEPCore) | [Link](https://github.com/adobe/aepsdk-core-ios) |
+| Extension | App Extension | CocoaPods | GitHub |
```

Comment:
Since you added this column here, you're going to need to add the extra column to the other rows that aren't affected, otherwise it will break.